src/containers/DebitCard/bridgeCard/zeroRestrictionMode/zeroRestrictionModeConfirmationModal.tsx

```
@@ -26,27 +26,30 @@ const styles = StyleSheet.create({
-      return 15;
+      return 60;
     case 1:
-      return 30;
+      return 720;
     case 2:
-      return 45;
+      return 1440;
     case 3:
-      return 60;
+      return 10080;
+    case 4:
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)